pkg/libinit/root_linux.go

```diff
@@ -271,26 +271,25 @@ func NewInitModuleLoader() *InitModuleLoader {
 // Useful for modules that need to be loaded for boot (ie a network
 // driver needed for netboot). It skips over blacklisted modules in
 // excludedMods.
-func InstallAllModules() error {
+func InstallAllModules() {
```

Comment:
May I suggest that this function signature not change and that it continue to return errors?  You can still ignore the returned errors in init_linux.go (and please add a comment into the code about why you're ignore the errors).

This way InstallAllModules() can be used by other code that might not want to ignore errors.

Thanks!

Reply:
Thats why I'm wondering if we should have a `ErrNoModulesFound` and a normal error. Then we can differentiate between the errors that gets returned.

Reply:
That sounds good. But if you return the error, how could you proceed further ?

Reply:
or maintain a list of errors ? -- if run into an error, append it, and in the end return it anyway.

Reply:
let's wait until go 1.20 for a list of errors, that stuff all gets better soon. 

And let's make our own ErrNoModulesFound for now?